qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -531,9 +532,10 @@ def main():
 
     # setup command line arguments
     parser = argparse.ArgumentParser(description='Mantid Workbench')
-    parser.add_argument('-x', '--execute', metavar='SCRIPT', dest='exe_script',
+    parser.add_argument('script', nargs='?')
```

Comment:
Actually I could change it so you must specify `-x` for each script you want to execute, otherwise it just opens it?
So `launch_mantidworkbench.sh -x script1.py script2.py -x script3.py` would execute script 1 and 3 and just open 2.

Reply:
It's better to leave it as one script.